src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -42,6 +42,7 @@ public AddCommand parse(String args) throws ParseException {
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+        //UniquePolicyList policyList = new UniquePolicyList();
```

Comment:
Can I check what is this for too?

Reply:
This is for when Anun implements `Policy` class into `Client` class!